pkg/storage/mountpoint.go

```diff
@@ -24,29 +23,41 @@ func (s *MountPointStorage) Mount(localPath string) error {
 	// NOTE: this is called to force unmount previous mounts
 	// It seems like mountpoint doesn't clean up gracefully by itself
 	s.Unmount(localPath)
-	os.MkdirAll(localPath, 0755)
+	if _, err := os.Stat(localPath); os.IsNotExist(err) {
+		err := os.MkdirAll(localPath, 0755)
+		if err != nil {
+			return err
+		}
+	}
+
+	endpoint := ""
+	if s.config.BucketURL != "" {
+		endpoint = fmt.Sprintf("--endpoint-url=%s", s.config.BucketURL)
+	}
 
+	// FIXME: cache? https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md#caching-configuration
```

Comment:
We could give users the option to toggle caching on this mount. This removes read-after-write consistency guarantees, but would likely be useful in heavy read situations. 

We could also expose the option to include the overwrite tag, which allows users to overwrite files in the volume. By default, any attempt to overwrite a file will result in an error.